e2e/autoscript-apps/src/authn-oauth/autoscript.ts

```diff
@@ -111,6 +116,24 @@ function autoscript() {
           return newTokens;
         },
       ),
+
+      rxDelay(delay),
+      mergeMap(
+        (tokens) => {
+          console.log('Proactively refresh tokens if expiring soon');
+          rxDelay(delay);
+          return forgerock.TokenManager.getTokens();
```

Comment:
I'm not an rxjs expert - and I think this is technically correct - but my functional brain tells me that this type should be an observable.

While I know promises and observables interop, I think this should be an `of(forgerock.TokenManager.getTokens()`. 

It may be a moot point, i'm not 100% sure so maybe @cerebrl can correct me here. I just know that typically monadic interfaces return the same context (observable) in this case.

Reply:
This is following a pattern heavily used throughout our autoscripts, so I wouldn't worry about it. But yes, Ryan, you are likely correct from a technical sense. From a practical one, this works without issue.

Reply:
Ok, I understand what you mean now. I went with the pattern throughout the rest of the tests, I thought it's probably best to be consistent if this works.